src/Cli/dotnet/CommandFactory/CommandResolution/CompositeCommandResolver.cs

```diff
@@ -1,10 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.DotNet.Cli.Utils;
+
 namespace Microsoft.DotNet.CommandFactory
 {
     public class CompositeCommandResolver : ICommandResolver
     {
+        private const string CommandResolveEvent = "dotnet/cli/commandresolution/commandresolved";
```

Comment:
you can remove the `dotnet/cli` prefix from the event name - the `TrackEvent` method adds this as a namespace to any emitted events.